src/controller/response/invoice-response.ts

```diff
@@ -22,6 +22,7 @@ import { BaseUserResponse } from './user-response';
 import { TransferResponse } from './transfer-response';
 import { PaginationResult } from '../../helpers/pagination';
 import { InvoiceState } from '../../entity/invoices/invoice-status';
+import { DineroObjectResponse } from './dinero-response';
 
 /**
  * @typedef {object} InvoiceStatusResponse
```

Comment:
This typedef name does not match the name of the object below.

Reply:
?